bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -489,13 +489,21 @@ async def _nomination_to_string(self, nomination: Nomination) -> str:
         entries_string = "\n\n".join(entries)
 
         start_date = time.discord_timestamp(nomination.inserted_at)
+
+        if nomination.thread_id:
+            thread = await self.bot.fetch_channel(nomination.thread_id)
+            thread_jump_url = f'[Jump to thread!]({thread.jump_url})'
+        else:
+            thread_jump_url = "**Thread hasn't been created yet or has been deleted**"
```

Comment:
If the thread was deleted `thread_id` would still be set, so I think we can just say `*Not created*` here, so it reads something like `Vote thread: *Not Created*`